ai2thor/platform.py

```diff
@@ -0,0 +1,238 @@
+import Xlib.display
+import glob
+import warnings
+import os
+import ctypes.util
+import xml.etree.ElementTree
+
+
+class Request:
```

Comment:
Can we give this a slightly more descriptive name? Perhaps `PlatformRequest` or `BuildRequest`?

Reply:
My argument for using just the name `Request` as opposed to `PlatformRequest` is that the class resides within the `ai2thor.platform` module.  When we use this class, we use it as `ai2thor.platform.Request` (within controller.py) so there shouldn't be any ambiguity about meaning.  As well, this is consistent with how the Python standard library names classes. Consider [Request](https://docs.python.org/3/library/urllib.request.html#request-objects).